website/docs/workflows/how-to-guides/set-up-automatic-processing.md

```diff
@@ -19,58 +19,56 @@ tags={[
+          const orderDetails = await getOrderDetails.run({ "order_id": order.order_id });
+          // Check if the order meets processing criteria
           if (orderDetails && orderDetails.amount < 10) {
-            // Add logic for processing if any
-            // initiates a refund
+            // Process refund
             await initiateRefund.run({
               "order_id": order.order_id,
               "status": 'Refund Processed'
             });
-            // Notifies the user about the processing
+            // Notify the customer
             await notifyUser.run({
-              "user_email": orderDetails.customer_email ,
+              "user_email": orderDetails.customer_email,
               "user_name": orderDetails.customer_name
             });
           }
         }
       }
-    }                              
+    }
     ```
 
-4. Trigger the workflow to automatically process requests. You can trigger the workflow in one of the following ways:
-    * If triggering the automated processing from within the Appsmith app, create a workflow query using **Trigger Workflow** as a request type, pass the parameters needed for processing, and bind it to the action from where the triggering happens. For more information, see [Trigger workflow from Appsmith app](/workflows/how-to-guides/trigger-workflow-from-appsmith-app).
-    * If triggering the automated processing from an external application, configure the workflow as a webhook trigger, pass the parameters in the body for processing, and use the workflow URL to trigger the workflow. For more information about triggering workflow from an external app, see [Trigger workflow using Postman](/workflows/tutorials/create-workflow#send-email-using-postman).
-6. Click the **Publish** button in the top right corner to apply your changes.
-7. Execute the workflow whenever a relevant event occurs. For example, a user submits a new request.
+4. Trigger the workflow to automatically process requests. You can do this in two ways:
+    * **Within an Appsmith app:** Create a workflow query with **Trigger Workflow** as the request type. Pass in the parameters for processing, and link it to the triggering action. Learn more in [Trigger Workflow from Appsmith App](/workflows/how-to-guides/trigger-workflow-from-appsmith-app).
+    * **From an external system:** Configure the webhook trigger in the workflow, Call the webhook through a POST request and pass the required parameters in the request body. Learn more in [Trigger Workflow using Postman](/workflows/tutorials/create-workflow#send-email-using-postman).
+
+6. Click **Deploy** in the top right to apply your changes.
+7. Execute the workflow when relevant events occur, like a new order request.
 
 ## Troubleshooting
 
-If you face issues, contact the support team using the chat widget at the bottom right of this page.
+For assistance, contact support via the chat widget at the bottom right.
```

Comment:
This is standard messaging that we are using across docs. Let's keep this as is.
```suggestion
If you face issues, contact the support team using the chat widget at the bottom right of this page.
```